### PR TITLE
Feat/voting balance improvements

### LINKED ISSUE #167: User cannot close account modal when clicking "X" button
**Environment**
Operating System: macOS 13.5.1 
Chrome browser: Version 116.0.5845.140
stCelo version: Can't find version (but last commit on main is [f13e722](https://github.com/celo-org/staked-celo-web-app/commit/f13e722f93e7360a0c5dbb01828294ad0cd2b6c9))
Wallet: Safe wallet (app.safe.global) connected using WalletConnect

**Describe the bug**
User cannot close "account" modal when clicking "X" button

**To Reproduce**
1. Go to app.stcelo.xyz
2. Click on the address
    
4. "Account" modal opens up
    

6. Click "X" button (top right)
7. Modal doesn't close

**See recording**

https://github.com/celo-org/staked-celo-web-app/assets/46296830/d922068c-81c1-44fb-9320-3541d2bdf8f0

**Expected behavior**
When clicking X, the modal should close. 

Current work around: Clicking anywhere outside the modal closes the modal.

### LINKED ISSUE #168: User cannot disconnect account
Reproduce the bug:

- I connected a Safe wallet (app.safe.global) using WalletConnect. 
- I can't find a button to disconnect the wallet in the stcelo UI:
  

- I disconnected in the Safe wallet, but the wallet is still connected in stcelo
- Re-loading the page doesn't fix this, the account is still connected.

### LINKED ISSUE #171: Make the buttons for retrieving locked funds more obvious

### LINKED ISSUE #172: Add a disclaimer when unstaking
Add a disclaimer for the balance showed when unstaking explaining why the balance may not be whole, because some is locked in voting.

### PR DESCRIPTION
**Pull request by nicolasbrugneaux**
This fixes various issues listed below 

- Closes #172
- Closes #171
- Closes #167

### CODE REVIEW

**Review by yerdua — APPROVED** [review]

**Comment by yerdua** [comment]
The disclaimer also shows up on the "Stake" tab. Is this intentional, or a bug?

image

File: src/features/swap/components/SwapForm.tsx [hunk 1]
```diff
@@ -151,7 +151,15 @@ const SwapFormInput = ({
...
-      infoChild={<BalanceTools mode={mode} onClickUseMax={setMaxAmount} balance={balance} />}
+      infoChild={
+        <div>
+          <BalanceTools mode={mode} onClickUseMax={setMaxAmount} balance={balance} />
+          <i className="text-[10px] text-color-secondary relative">
+            <span className="absolute left-[-0.5rem] top-0">*</span>
+            Only unlocked funds are shown above. For Locked funds see the Govern tab.
+          </i>
+        </div>
+      }
```
Comment (yerdua) [inline src/features/swap/components/SwapForm.tsx:162]: should there also be a check for the mode here, similar to what's going on in BalanceTools? `mode === Mode.unstake`

**Review by yerdua — CHANGES_REQUESTED** [review]
I'm guessing that the appearance of the disclaimer on the `Stake` tab is unintentional, as the star only shows up next to the balance for `Unstake`

File: src/features/swap/components/SwapForm.tsx (hunk 1 above)
Comment (nicolasbrugneaux) [reply src/features/swap/components/SwapForm.tsx:162]: Woopsie doopsie. Great catch.

**Review by yerdua — APPROVED** [review]

**Pull request merged**